back/app/models/admin_publication.rb

```diff
@@ -87,6 +87,14 @@ def published?
     publication_status == 'published'
   end
 
+  def ever_published?
```

Comment:
Do we need both ever and never published defined? Personal preference so def not blocking but I would keep only one of these.

Reply:
![giphy](https://github.com/user-attachments/assets/7f149fe6-18d4-47af-8ddc-89862f593bea)